app/javascript/controllers/index.js

```diff
@@ -28,6 +28,9 @@ application.register("form", FormController)
 import MessageDraftsController from "./message_drafts_controller"
 application.register("message-drafts", MessageDraftsController)
 
+import FsMessageDraftsController from "./fs_message_drafts_controller"
+application.register("fs-message-drafts", FsMessageDraftsController)
```

Comment:
Tu by som povedal, ze to volajme dropzonecontroller, lebo s FS message tam nic nevidim a mozeme to pouzivat aj inde pripadne.